Khan/live-editor
# Make more strings translatable

### High-level description of change

The strings in Handlebars templates aren't translatable by webapp, so this moves the rest of them out. I did not move them out of live-editor.handlebars, as webapp does not use that file, it overrides the rendering with a React output.

I also fixed an issue with an untranslatable SQL error message. 

### Are there performance implications for this change?

No

### Have you added tests to cover this new/updated code?

No tests added. I ran existing tests, which includes a test for SQL error messages. No tests for the UI, so I manually tested those.

### Risks involved

I'm most nervous about the SQL change, as it assumes the format of the string. However, I checked the C code that it's compiled from, and the format seems consistent.

### Are there any dependencies or blockers for merging this?

No

### How can we verify that this change works?

- Open the SQL demo, see results, remove a semi-colon, see error
- Open the webpage demo, type an <img tag>, click for file picker, see "Pick file", click for modal, see "OK" and close" in modal
- Open the PJS demo, type playSound(getSound("rpg/metal-clink"));, click for file picker, see "Pick file", click for modal, see "OK" and close" in modal

Note that the modal doesn't work in this repo, it only works in webapp. Apparently we never included bootstrap-modal here.
